actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/responsecode/ResponseCode.java

```diff
@@ -834,6 +834,9 @@
   unableToConnectToAdminUtil(
       ResponseMessage.Key.UNABLE_TO_CONNECT_TO_ADMINUTIL,
       ResponseMessage.Message.UNABLE_TO_CONNECT_TO_ADMINUTIL),
+  unableToConnectToFormApiUtil(
+      ResponseMessage.Key.UNABLE_TO_CONNECT_TO_ADMINUTIL,
```

Comment:
Admin_util is used insted of form util response msg

Reply:
I will remove it.. as it is not being used..as instead of throwing exceptions